external-crates/move/crates/move-vm-runtime/src/loader/translate2.rs

```diff
@@ -0,0 +1,874 @@
+// Copyright (c) The Diem Core Contributors
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Since this file demonstrates the _next_ step of the translation, please delete it and let's stack it in the next PR. That way we don't get a lot of weird code motion between PRs.

Reply:
Not sure if that's possible -- it's needed as part of the loading process for packages (or at least to be able to actually test the loading process). I can try and merge this into `translate.rs`, but again there might be issues with that since this is expecting/using a very different context which the old model will have problems filling in correctly.

Reply:
Just leave it separate, then. It's just strange to leave it here temporarily, but since the next PR should get rid of it, it seems okay enough.